packages/algob/src/index.ts

```diff
@@ -1,4 +1,5 @@
 import {
+  getmultisigAddress,
```

Comment:
this function name should be `getMultisigAddress`. Please rename

Reply:
updated.